explainaboard/processors/processor_factory.py

```diff
@@ -66,6 +69,8 @@
     TaskType.tabular_classification: TabularClassificationProcessor,
     TaskType.argument_pair_extraction: ArgumentPairExtractionProcessor,
     TaskType.meta_evaluation_nlg: MetaEvaluationNLGProcessor,
+    TaskType.argument_pair_identification: ArgumentPairIdentificationProcessor,
+    TaskType.ranking_with_context: ArgumentPairIdentificationProcessor,
```

Comment:
This line seems incorrect? Because there's no general purpose `ranking_with_context` processor yet it'd probably be better to just delete this line and allow an error to be thrown if someone tries to specify this task.

```suggestion
```

Reply:
thanks a lot for fixing the docs lint issue I also fix this comment!